src/sourmash/command_compute.py

```diff
@@ -326,7 +326,16 @@ def save_sigs_to_location(siglist, save_sig):
 class ComputeParameters(RustObject):
     __dealloc_func__ = lib.computeparams_free
 
-    def __init__(self, ksizes, seed, protein, dayhoff, hp, dna, num_hashes, track_abundance, scaled):
+    def __init__(self,
+                 ksizes=(21, 31, 51),
```

Comment:
do we want to put a `*` in here? it would break some more code but would be good.

Happy to submit my own PR to do this after this is merged, if you like.

Reply:
Good point. I wanted the `__repr__` to make it easier to figure out what values were being passed in the protein case, and explicit values here to make sure they matched the defaults in Rust.

(it is funny that you also implemented `__repr__` recently =P)

Reply:
I put the `*` in the latest commit, but technically that's a breaking change in the API :see_no_evil:

Reply:
(but agree it is a good thing to put the `*`)

Reply:
I have generally refrained from putting any guarantees on APIs that aren't exposed at the top level sourmash namespace, FWIW. Well, and the CLI.